src/main/bash/sdkman-install.sh

```diff
@@ -150,7 +151,8 @@ function __sdkman_download() {
 		echo ""
 
 		# download binary
-		__sdkman_secure_curl_download "${download_url}" --output "${binary_input}" --dump-header "${headers_file}"
+		__sdkman_secure_curl_download "${download_url}" --output "${binary_input}" --dump-header "${tmp_headers_file}"
+		grep '^X-Sdkman' "${tmp_headers_file}" > "${headers_file}"
```

Comment:
Don’t recall if the message key is spelled SdkMan or Sdkman, maybe do grep -i to ignore case?

Reply:
I checked in the broker service and subsequently tested it with Gradle 2.11 that you uploaded with a checksum. Works perfectly. Also, all our checksum validation tests are still green.

Reply:
Should we delete the temp file as well?

Reply:
I thought I'd leave it for `flush` to clean up. That way we can go back and trace if something goes wrong.